### PR TITLE
Add locust

### PR DESCRIPTION
**Pull request by anayden**

### CODE REVIEW

File: .neuro/live.yaml [hunk 1]
```diff
@@ -7,36 +7,36 @@ defaults:
 
 volumes:
   data:
-    remote: storage://neuro-compute/yevheniisemendiak/ml_recipe_bone_age/data
+    remote: storage://onprem-poc/alexeynayden/data
```
Comment (yartem) [inline .neuro/live.yaml:10]: please use `storage:ml_recipe_bone_age/data` -- it's generalized values (for neuro-compute) that we will introduce in https://github.com/neuro-inc/ml-recipe-bone-age/pull/18

File: .neuro/live.yaml [hunk 2]
```diff
@@ -238,15 +239,17 @@ jobs:
   deploy_inference_platform: # todo: Alexey is working on it
     image: $[[ images.seldon.ref ]]
     name: $[[ flow.title ]]-test-inference
-    preset: gpu-k80-small-p
+    preset: gpu-small
```
Comment (yartem) [inline .neuro/live.yaml:242]: again, this job is supposed to run on neuro-compute, so `gpu-k80-small-p`

File: requirements.txt [hunk 3]
```diff
@@ -7,3 +7,5 @@ numpy>=1.17.*
 matplotlib>=3.1.*
 torchvision==0.8.2
 mlflow==1.13.1
+azure-storage-blob==2.1.0
+locust==1.4.1
```
Comment (yartem) [inline requirements.txt:11]: which image uses it? :/

Comment (anayden) [reply requirements.txt:11]: No image does, I added this to build my local virtualenv for testing, but forgot to remove.

**Pull request merged**